src/Pages/gyuyoung/Login/Login.js

```diff
@@ -1,181 +1,93 @@
-이벤트 타겟이 미적용되는거 확인
-
-[event.target.name]설정법 공부할것
-좀더 효율적인 함수 사용 조사하고 정리할것
-*/
-  handleChange1=(event)=>{
-    this.setState({Id : event.target.value});
-
-    /*
-    원하는 결과가 안됨
 
-    스테이트에서 value를 가져오면 하나씩 짤린다??
+InPutChangeHandler=(event)=>{
+  this.setState({[event.target.name] : event.target.value});
+  if(event.target.name === 'id'){
+    if(event.target.value.length>3 && this.state.pw.length>8){
+      this.setState({loginBtnClass : 'loginButton loginBtnColorChanged'})}
+    else{this.setState({loginBtnClass : 'loginButton loginBtnColor'})}
 
-    문제 확인 스테이트에 value는 함수가 종료된뒤에 update된다
-    그러면 어떻게해야 제대로된 value를 챙길수있을까?
-
-    event 타겟 value는 자체가 온전함
-
-    문제는 구분을 못시킨다
-    control에서 문제가 생겼음
-
-    답찾았다!!!!!!
-    인자를 2개 따로따로 받아냈다
-
-    좀더 효율적인 value활용법 필요함
-    */
-    this.LoginBtnControl(event.target.value, this.state.Pw);
-    this.IdInputControl(event);
-  }
-  handleChange2=(event)=>{
-    this.setState({Pw : event.target.value});
-    this.LoginBtnControl(this.state.Id, event.target.value);
-    this.PwInputControl(event);
+    if(event.target.value.length>0){
+        this.setState({idInInputBoxPlaceholderChanged : 'changedInInputBoxPlaceholder'})
+    }
+    else{this.setState({idInInputBoxPlaceholderChanged : 'inputBoxPlaceholder'})}
   }
+  else{
+    if(event.target.value.length>8 && this.state.id.length>3){
+      this.setState({loginBtnClass : 'loginButton loginBtnColorChanged'})}
+    else{this.setState({loginBtnClass : 'loginButton loginBtnColor'})}
 
-  LoginBtnControl=(Id, Pw)=>{
-    if(Id.length >= 3 && Pw.length >= 8){
-        this.setState({Btn : '#2a96f6'});
+    if(event.target.value.length>0){
+      this.setState({pwInInputBoxPlaceholderChanged : 'changedInInputBoxPlaceholder'})
+      if(this.state.show === ''){
+        this.setState({show : 'Show'})
+        
+      }
     }
     else{
-        this.setState({Btn : '#b9dcfd'});
+      this.setState({show : '',type : 'password',pwInInputBoxPlaceholderChanged : 'inputBoxPlaceholder'})
     }
   }
-/*
-  스타일에는 어떻게 관여하지??
-  
-  스타일에 직접 관여하는법과 클레스네임 변경이 있는건 안다.
-  문제는 네임변경법보단 직접 관여해서 컨트롤해보고싶은데...
-
-  대괄호도 아니다...
-
-  이부분은 블로깅 다시 정리해볼것
-
-  부드러운 애니매이션 효과가 없다????
-  뭐가 문제인지 아직 미확인
-*/
-  IdInputControl=(event)=>{
-      if(event.target.value.length >= 1){
-        this.setState({fontId : '12px', transformId : 'scale(.83333) translateX(-23px) translateY(-10px)'})
-      }
-      else{this.setState({fontId : '14px', transformId : ''})}
-  }
-  
-  PwInputControl=(event)=>{
-      if(event.target.value.length >= 1){
-        this.setState({fontPw : '12px', transformPw : 'scale(.83333) translateX(-23px) translateY(-10px)'})
-      }
-      else{this.setState({fontPw : '14px', transformPw : '', show : ''})}
+}
 
-      if(this.state.show === ''){
-          this.setState({show : 'Show'})
-      }
+changePwType=()=>{
+  if(this.state.show === 'Show'){
+    this.setState({type : 'text', show : 'Hide'})
   }
+  else{this.setState({type : 'password', show : 'Show'})}
+}
```

Comment:
일단 다음과 같이 띄어쓰기를 적절히 활용하여 보기 좋게 만드는 연습을 하시는게 좋겠습니다. 
```suggestion
changePwType = () => {
  if (this.state.show === 'Show') {
    this.setState({ type : 'text', show: 'Hide' })
  } else {
    this.setState({ type : 'password', show : 'Show' })
  }
}
```

다음은 삼항연산자를 활용하여 조건문을 간결하게 만드는 방법입니다. 기본적으로 setState는 하나의 함수에서 한번만 사용한다는 생각으로 로직을 작성해주시면 좋습니다.
```suggestion
changePwType = () => {
  const isShow = this.state.show === "Show"
  this.setState({
    type : isShow ? "text" : "password",
    show : isShow ? "password" : "show"
  })
}
```